UC4-chaincode/src/test/resources/test_configs/group_contract/AddUserToGroupTestIO.json

```diff
@@ -0,0 +1,313 @@
+[
+  {
+    "name": "addNonExistingUserToGroup",
+    "type": "addUserToGroup_SUCCESS",
+    "ids": [
+      {
+        "id": "User1",
+        "type": "admin"
+      },
+      {
+        "id": "0000001",
+        "type": "member"
+      }
+    ],
+    "setup": {
+      "certificateContract": [
+        "0000001",
+        "test-certificate"
+      ]
+    },
+    "input": [
+      "0000001",
+      "0123"
+    ],
+    "compare": [
+      {
+        "groupId": "0123",
+        "userList": [
+          "0000001"
+        ]
+      }
+    ]
+  },
+  {
+    "name": "addDuplicateUsersToOneGroup",
+    "type": "addUserToGroup_SUCCESS",
+    "ids": [
+      {
+        "id": "User1",
+        "type": "admin"
+      },
+      {
+        "id": "0000001",
+        "type": "member"
+      }
+    ],
+    "setup": {
+      "groupContract": [
+        "0123",
+        {
+          "groupId": "0123",
+          "userList": ["0000001"]
+        }
+      ],
+      "certificateContract": [
+        "0000001",
+        "test-certificate"
+      ]
+    },
+    "input": [
+      "0000001",
+      "0123"
+    ],
+    "compare": [
+      {
+        "groupId": "0123",
+        "userList": [
+          "0000001"
+        ]
+      }
+    ]
+  },
+  {
+    "name": "addSecondUserToGroup",
+    "type": "addUserToGroup_SUCCESS",
+    "ids": [
+      {
+        "id": "User1",
+        "type": "admin"
+      },
+      {
+        "id": "0000002",
+        "type": "member"
+      }
+    ],
+    "setup": {
+      "groupContract": [
+        "0123",
+        {
+          "groupId": "0123",
+          "userList": ["0000001"]
+        }
+      ],
+      "certificateContract": [
+        "0000002",
+        "test-certificate"
+      ]
+    },
+    "input": [
+      "0000002",
+      "0123"
+    ],
+    "compare": [
+      {
+        "groupId": "0123",
+        "userList": [
+          "0000001",
+          "0000002"
+        ]
+      }
+    ]
+  },
+  {
+    "name": "addUserToTwoGroups",
```

Comment:
Unfitting name. User is already in two groups, right?

Reply:
```suggestion
    "name": "addDuplicateUserToTwoGroups",
```